tools/fdleak.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/python
```

Comment:
You missed another two files(README.md, man8), please check Brendan's lead
https://github.com/brendangregg/bcc/commit/9fa156273b395cfc5505f0fff5d6b7b1396f7daa